server/views/pages/bookVisit/additionalSupport.njk

```diff
@@ -1,4 +1,4 @@
-{% extends "../../partials/layout.njk" %}
+{% extends "partials/layout.njk" %}
```

Comment:
Noticed that we had a mix of Nunjucks import paths. Updated all to use the simplest form.